### ISSUE TITLE
The bdist should not contain the docs and tests

### ISSUE THREAD

**Issue by bmbouter** [pulp/pulpcore#2892]
Our setup.py is not instructing setuptools correctly. The issue is likely [in this line](https://github.com/pulp/pulpcore/blob/main/setup.py#L16). See the [setuptools docs](https://setuptools.pypa.io/en/latest/userguide/package_discovery.html) for how to fix.

I confirmed this by building the bdist locally, e.g. `python setup.py bdist`.

**Comment by dralley** [comment]
The issue historically, at least with tests, is that unit tests are run with the django test runner (which is needed to set up and tear down a temporary database), which requires them to be in the same package.

**Comment by bmbouter** [comment]
> The issue historically, at least with tests, is that unit tests are run with the django test runner (which is needed to set up and tear down a temporary database), which requires them to be in the same package.

Makes sense. My claim is that they shouldn't be in the bdist, only in the sdist. So if you're installing a bdist, you wouldn't be able to run any of the tests.

**Comment by fao89** [comment]
https://github.com/pulp/plugin_template/pull/680
